byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -52,4 +71,44 @@ private String formatLocation(PlaceType location) {
     place = place.replace('_', ' ');
     return place;
   }
+
+  /**
+   * Temporary setup to find entity with hard-coded id value. Set their interests.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String tripIdString = request.getParameter("tripId");
+    Long tripId = Long.parseLong(tripIdString);
```

Comment:
I actually changed this entirely since we are passing in the key as a string directly, rather than the id to said key. This avoids converting to id, then creating a new key. Instead, we pass the key as string, then convert that string into the key directly